cfp.md

```diff
@@ -1,9 +1,64 @@
 ---
 layout: page
-title: "BSidesSF 2019 Call For Papers"
+title: "BSidesSF 2020 Call For Participation"
 permalink: /cfp.html
+redirect_from: "/cfw"
+redirect_from: "/cfw.html"
```

Comment:
This is where the `jekyll-redirect-from` plugin magic is invoked, just in case someone happened to have the CFW pages bookmarked.